src/canmatrix/canmatrix.py

```diff
@@ -1227,43 +1417,71 @@ def delSignal(self, signal):
                 for sig in signalList:
                     frame.signals.remove(sig)
 
-    def addSignalReceiver(self, frameName, signalName, ecu):
-        frames = self.globFrames(frameName)
+    def addSignalReceiver(self, framePattern, signalName, ecu):
+        """Add Receiver to all Frames by name pattern.
+
+        :param str framePattern: Frame name pattern
+        :param str signalName: signal name
+        :param str ecu: Receiver ECU name
+        """
+        frames = self.globFrames(framePattern)
```

Comment:
If it's going to be exclusively a glob I'd use that in the name rather than the generic 'pattern'.  Ditto for the docstring.  Glob, like regex, isn't a single thing but it's a _relatively_ standard pattern so I'd think it's useful to be explicit.

May as well catch `signalName` as well if we're going to exclusively glob it.  Also, find all similar below (or move them to their own PR).